### DISCUSSION TITLE
What is the lowest downgrade version available for kernver 2?

### DISCUSSION THREAD

**Discussion by FunnyCorgi** [3kh0/ext-remover#1088] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I would like to know how low I can go with downgrading. I want to find out because I found a way to flash it, and want to know the lowest possible version.

**Comment by helloIlovegimkit** [comment, marked as answer]
There isn't one I'm pretty sure though if there is the best one to downgrade to is v115-v119

**Comment by gomes-web** [comment]
go to this link 

> https://brandon421-ops.github.io/KVS/#:~:text=kernver%202%3A%20112%20%2D%20119
